src/components/transition/classNamesRegistry.js

```diff
@@ -0,0 +1,22 @@
+// @flow strict
+
+export interface ClassNamesRegistryType {
+  register(key: string, value: string): void;
+  toString(): string;
+}
+
+export function createClassNamesRegistry(): ClassNamesRegistryType {
```

Comment:
Where is this function being used?

Reply:
Sure, i just wasn't able to find any import of this function, neither in examples nor in tests.

Reply:
ok I see it, diff was hidden loaded, sorry